timesketch/frontend-ng/src/components/Explore/AggregateDialog.vue

```diff
@@ -0,0 +1,537 @@
+                  <li>Selected Field:&nbsp;<span style="font-family: monospace">{{  this.eventKey }}</span></li>
+                  <ul>
+                    <li>Count: {{ this.fieldValueCount }}</li>
+                    <li>Unique: {{ this.fieldCardinality }}</li>
+                  </ul>
+                </ul>
+                <br>
+                <ul>
+                  <li>Selected Value:&nbsp;<span style="font-family: monospace">{{  this.eventValue }}</span></li>
+                  <ul>
+                    <li>Count: {{ this.valueEventCount }}</li>
+                    <li>Min date: {{ this.valueDateTimeMinimum }}</li>
+                    <li>Max date: {{ this.valueDateTimeMaximum }}</li>
+                  </ul>
+                </ul>
+
+              </v-card-text>
+            </v-card>
+          </v-col>
+          <v-col align="center">
+            <v-card height="450px" :loading="!statsReady">
+              <v-card-title>
+                Count of &nbsp;<span style="font-family: monospace">{{ eventValue }}</span>&nbsp; as a percentage of &nbsp;<span style="font-family: monospace">{{ eventKey }}</span>&nbsp; events
+              </v-card-title>
+              <v-card-text v-if="statsReady">
+                <apexchart
+                  height="250px"
+                  :options="this.donutChartOptions"
+                  :series="this.donutChartSeries"
+                ></apexchart>
+              </v-card-text>
+            </v-card>
+          </v-col>
+
+          <v-col align="center">
+            <v-card height="450px" :loading="!statsReady">
+              <v-card-title>
+                Event distribution by {{ this.distributionIntervals[this.selectedDistributionIntervalIndex] }}
+              </v-card-title>
+              <v-card-subtitle>
+                Selected value:&nbsp;<span style="font-family: monospace">{{ eventValue }}</span>&nbsp;
+              </v-card-subtitle>
+              <v-card-text v-if="statsReady">
+                <v-btn-toggle mandatory v-model="selectedDistributionIntervalIndex">
+                  <v-btn v-for="interval in this.distributionIntervals" :key="interval" small>
+                    {{ interval }}
+                  </v-btn>
+                </v-btn-toggle>
+                <apexchart
+                  height="300px"
+                  :options="this.intervalChartOptions"
+                  :series="this.intervalChartSeries"
+                ></apexchart>
+              </v-card-text>
+            </v-card>
+          </v-col>
+        </v-row>
+        <v-row>
+            <v-col align="center">
+              <v-card height="500px" :loading="!statsReady">
+                <v-card-title>
+                  Top {{ Math.min(10, this.commonValues.length) }} &nbsp;<span style="font-family: monospace">{{ eventKey }}</span>&nbsp; values (out of {{ this.fieldCardinality }})
+                </v-card-title>
+                <v-card-text>
+                  <v-data-table
```

Comment:
10 is the default, we can return more. What about returning all and keep the pagination?

Reply:
Both works for me, as long as we don't call it Top 10 anymore if we go with all entries.

Reply:
Done (added the pagination footer check).

For the top/most common values, the results will be limited to 10 since that is an open search parameter for this aggregation (which defaults to 10).  Because of this, I would opt to leave it to 10 (or similar number) for a few reasons:
* returning "all" will introduce additional logic to split between "common" and "rare"
* for certain numeric and text fields, the number of values could be very high and it wouldn't be useful to navigate these values "10 at a time".  
 
For the rare values, the results are limited by document count (i.e. number of events).  This is currently set to maximum of 5 so in some cases the number of results could vary significantly.

Reply:
Given top 10 is the default and should cover most needs for an analyst I will resolve this comment. We can revisit the default value or make them even choosable by the analyst in the next iteration.